server/index.js

```diff
@@ -0,0 +1,35 @@
+// @ts-check
```

Comment:
Very good. Ports are documented as expected.
All the initial functionalities are included.